style.css

```diff
@@ -242,6 +242,74 @@ hr {
   color: white;
 }
 
+.contact {
+  display: flex;
+  flex-direction: column;
+  background: #6071fe;
+  justify-content: center;
+  align-items: center;
+  padding: 24px;
+  border-radius: 50px 0 0;
+}
+
+.contact-title {
+  font-weight: 600;
+  font-size: 2em;
+  color: white;
+}
+
+.contact-subtitle {
+  font-weight: 100;
+  font-size: 1.2em;
+  text-align: center;
+  color: #d9dbff;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.form-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-radius: 0 50px 0 0;
+}
+
+input {
+  width: 150%;
+  border-radius: 2px;
+  border: 1px solid #6071fe;
+  padding: 10px;
+  margin-bottom: 10px;
+}
+
+input:focus {
+  outline: none;
+}
+
+textarea {
+  width: 150%;
+  height: 100px;
+  border-radius: 2px;
+  border: 1px solid #6071fe;
+  padding: 10px;
+  margin-bottom: 10px;
+  resize: none;
+}
+
+.submit {
+  width: 100%;
+  border-radius: 10px;
+  border: 1px solid #6071fe;
+  padding: 10px;
+  color: #6071fe;
+  margin-top: 10px;
+  background: white;
+  font-weight: bold;
+  font-size: medium;
+  cursor: pointer;
+}
+
```

Comment:
- [ ] Please try to  follow the Figma design Requirement, You must stick to the design as much as possible this includes the font, colors, images and spacing (margin/padding) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
- [ ] Kindly implement Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
- [ ] kindly try to include background images just like Figma design show below : 
![fig12](https://user-images.githubusercontent.com/93598189/186223324-8cb4405d-4479-4512-bdcf-505c8bd8df99.PNG)

Your Project Look likes this : 

![cont](https://user-images.githubusercontent.com/93598189/186223465-17291297-79be-4622-89bb-3a63ff5942ec.PNG)